man/man8/zstreamdump.8

```diff
@@ -32,6 +33,12 @@ The following options are supported:
 Suppress the validation of checksums.
 .RE
 
+.ad
+.sp .6
+.RS 4n
+Only dump information about the resume token.
+.RE
```

Comment:
I think you can revert the changes to this file since it's now (only) in `zstream.8`, right?

Reply:
Yep.